src/sentry/quotas/redis.py

```diff
@@ -240,15 +269,15 @@ def is_rate_limited(self, project, key=None, timestamp=None):
 
             assert quota.should_track
 
-            shift = project.organization_id % quota.window
-            key = self.__get_redis_key(quota, timestamp, shift, project.organization_id)
-            return_key = self.get_refunded_quota_key(key)
-            keys.extend((key, return_key))
+            shift: int = project.organization_id % quota.window
+            quota_key = self.__get_redis_key(quota, timestamp, shift, project.organization_id)
+            return_key = self.get_refunded_quota_key(quota_key)
+            keys.extend((quota_key, return_key))
             expiry = self.get_next_period_start(quota.window, shift, timestamp) + self.grace
 
             # limit=None is represented as limit=-1 in lua
             lua_quota = quota.limit if quota.limit is not None else -1
-            args.extend((lua_quota, int(expiry)))
+            args.extend((lua_quota, int(expiry)))  # type: ignore[arg-type]
```

Comment:
I think this is pointing out that the type on `args` was incorrect -- notably this is using `extend` which acts like:

```python
>>> x = []
>>> x.extend((1, 2, 3))
>>> x
[1, 2, 3]
```

rather than append which would push items in -- I think args should be `list[int]` maybe?

Reply:
Yes, you're right